nix/overlays.nix

```diff
@@ -0,0 +1,40 @@
+self: final: prev: {
+  pythonPackagesOverlays =
+    (prev.pythonPackagesOverlays or [])
+    ++ [
+      (_: pprev: {
+        pywlroots = (pprev.pywlroots.overrideAttrs(_: rec {
```

Comment:
Pywlroots has been upstreamed, the overlay can be removed

Reply:
I have overlay just in case, when qtile moves on to 0.18

Reply:
We can just add it back when that happens

Reply:
Why delete tho?

Reply:
I don't like to have unnecessary code around

Reply:
Well, we can remove it ig